src/main/java/org/jboss/pnc/deliverablesanalyzer/model/FinderResultCreator.java

```diff
@@ -228,33 +225,35 @@ private static Build createBuild(
             builder.brewId((long) kojiBuild.getBuildInfo().getId());
             builder.brewNVR(kojiBuild.getBuildInfo().getNvr());
         }
+
         return builder.isImport(kojiBuild.isImport()).artifacts(artifacts).build();
     }
 
     private static Artifact createArtifact(KojiLocalArchive localArchive, BuildSystem buildSystem, boolean imported) {
         KojiArchiveInfo archiveInfo = localArchive.getArchive();
-        String buildType = archiveInfo.getBuildType();
+        KojiBtype buildType = archiveInfo.getBuildType();
         ArtifactBuilder<?, ?> builder;
 
         switch (buildType) {
-            case GRADLE, MAVEN, SBT -> builder = createMavenArtifact(archiveInfo);
-            case NPM -> builder = createNpmArtifact(archiveInfo);
+            case maven -> builder = createMavenArtifact(archiveInfo);
```

Comment:
what happened to the GRADLE and SBT type?

Reply:
I am adding a new Windows (`WIN`) build type to DelAn that already exists in kojiji in order to allow DelAn to scan zips that were created on a Windows host.

There never was a "gradle" or "sbt" type in Koji. This was a string field that was being "abused" in DelAn. I am changing the type to an enum in Kojiji to prevent this. See this PR https://github.com/release-engineering/kojiji/pull/167.

Ultimately, I don't think that this PR changes anything in DelAn since it always called `createMavenArtifact` for the "gradle" or "sbt" types. The extra information will still be available somewhere else, but not in this field.

Reply:
Ah, I see you are mapping it in buildfinder here https://github.com/project-ncl/build-finder/pull/1294/files#diff-f70b10b14f82c099497e76bf8a9bd558a1bb123b3390727f5a9c2ccf7a7057e1R309
Ok

Reply:
This change is made up of the two PRs above, plus https://github.com/project-ncl/pnc-api/pull/244. The kojiji and pnc-api PRs need to go in first.

The current mapping from PNC to Brew is now `if NPM then NPM else MAVEN`.

Brew supports `WIN`, `IMAGE` and `RPM` types which aren't supported by PNC. The "gradle" and "sbt" types were PNC types not supported by Brew.
 
I am adding the `WIN` type to PNC (DelAn only). I don't see how the `IMAGE` type is useful, and the `RPM` type is the default type, and I am not sure that we scan RPMs directly with DelAn (although it's possible to do).